test/dom/dom.ts

```diff
@@ -2,7 +2,7 @@ import {assert} from "chai";
 import JSBI from "jsbi";
 import * as ion from "../../src/Ion";
 import {IonTypes} from "../../src/Ion";
-import {Value} from "../../src/dom";
+import {Value, load, loadAll} from "../../src/dom";
```

Comment:
References to `ion.load()` (a re-export of `ion.dom.load()`) in this file have been changed to direct references to `ion.dom.load()` to avoid a compile time issues that arise inconsistently.